src/main/java/org/jenkinsci/plugins/envinjectapi/util/EnvVarsResolver.java

```diff
@@ -183,7 +183,7 @@ public static String resolveEnvVars(@Nonnull Run<?, ?> run, @CheckForNull String
         if (node != null) {
             DescribableList<NodeProperty<?>, NodePropertyDescriptor> nodeProperties = node.getNodeProperties();
             for (NodeProperty nodeProperty : nodeProperties) {
-                if (nodeProperty != null && nodeProperty instanceof EnvironmentVariablesNodeProperty) {
+                if (nodeProperty instanceof EnvironmentVariablesNodeProperty) {
```

Comment:
IIRC it was for old FindBugs versions which did not handle it correctly

Reply:
I guess it will be optimized away anyhow. But it is clearer without the null check.